app/src/main/java/com/example/themoviedb/feature07/MoviesRetrofitClient.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.themoviedb.feature07
+
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+object MoviesRetrofitClient {
```

Comment:
This is the same object that will be used for People aswell. Naming it MoviesRetrofitClient might not be the best. A generic name for it would be ideal